hack/addons/update-cluster-autoscaler.sh

```diff
@@ -21,7 +21,16 @@ readonly FILE_NAME="cluster-autoscaler.yaml"
 
 readonly KUSTOMIZE_BASE_DIR="${SCRIPT_DIR}/kustomize/cluster-autoscaler/"
 envsubst -no-unset <"${KUSTOMIZE_BASE_DIR}/kustomization.yaml.tmpl" >"${ASSETS_DIR}/kustomization.yaml"
-cp "${KUSTOMIZE_BASE_DIR}"/*.yaml "${ASSETS_DIR}"
+cat <<EOF >"${ASSETS_DIR}/gomplate-context.yaml"
```

Comment:
I don't think I understand what this is doing.
Can you please add a comment to leave some hints for future us why this is needed and why this is different from the other scripts.

Reply:
Added comment.